library/dnet_common.c

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright 2008+ Evgeniy Polyakov <zbr@ioremap.net>
+ * Copyright 2015+ Yandex
```

Comment:
Is there any suggestions from FSF or any other foundation how to add copyright lines to files under GPL license? I found that copyright holders should be mentioned in notice.